src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/RuleExporter.cs

```diff
@@ -27,6 +28,7 @@ private static class ProjectRules
             ///     for a project.
             /// </summary>
             [ExportRule(nameof(SupportedTargetFrameworkAlias), PropertyPageContexts.ProjectSubscriptionService)]
+            [ExportDesignTimeBuildTargets(nameof(SupportedTargetFrameworkAlias))]
```

Comment:
I wonder both SupportedTargetFrameworkAlias & CollectedFrameworkReference might be only used in VS, but not VS Green...